mmv1/products/filestore/terraform.yaml

```diff
@@ -26,10 +26,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           instance_name: "test-instance"
       - !ruby/object:Provider::Terraform::Examples
         name: "filestore_instance_full"
-        min_version: beta
         primary_resource_id: "instance"
         vars:
           instance_name: "test-instance"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "filestore_instance_enterprise"
+        primary_resource_id: "instance"
+        skip_test: true
```

Comment:
Followed up offline, this was following an example pubsub_topic_cmek.tf.erb and because the test creates cmek keys that aren't managed by terraform and break the test on subsequent runs

Reply:
Built and tested it locally, but looks like I am having quota issues `Quota limit 'EnterpriseStorageGibPerRegion' has been exceeded. Limit: 0 in region us-central1.` Already request the quota increase.

Reply:
Tested it locally and tests passed!